user/enterprise/high-availability.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Enterprise High Availability Mode
+layout: en_enterprise
+
+---
+
+Travis CI Enterprise typically runs as a single container instance communicating with one or multiple workers and generally, this setup is quite stable and maintainable. However, in some cases -- especially volume licensing or by corporate requirement -- you might need to install Travis CI in High Availability (HA) mode. 
```

Comment:
Seems odd to say you might *need* without the reader knowing why they *might* need to? Or do they? If not maybe want is clearer?